tests/test_optimizer_api.py

```diff
@@ -0,0 +1,49 @@
+import time
+import unittest
+
+from fds.analyticsapi.engines.api.optimizations_api import OptimizationsApi
+from fds.analyticsapi.engines.models.axioma_equity_optimization_parameters import AxiomaEquityOptimizationParameters
+from fds.analyticsapi.engines.models.strategy import Strategy
+from fds.analyticsapi.engines.models.account import Account
+from fds.analyticsapi.engines.models.optimization import Optimization
+from fds.analyticsapi.engines.models.output_types import OutputTypes
+from fds.analyticsapi.engines.models.trades_list import TradesList
+
+import common_parameters
+from common_functions import CommonFunctions
+
+
+class TestOptimizerApi(unittest.TestCase):
+
+    def setUp(self):
+        self.api_client = CommonFunctions.build_api_client()
+
+    def test_normal_workflow(self):
+        optimizer_api = OptimizationsApi(self.api_client)
+        strategy = Strategy(id="Client:test")
+        account = Account(id="Client:/analytics/nyim/TEST.ACCT")
+        optimization = Optimization(riskmodeldate="0M", backtestdate="0M")
+        trade_list = TradesList(identifiertype="Asset", includecash=False)
```

Comment:
Can you please move these arguments to [common_parameters.py](https://github.com/factset/analyticsapi-engines-python-sdk/blob/master/tests/common_parameters.py)?

Reply:
Done